### PR TITLE
[TensorBoard] Fixing breaking changes in TensorBoard

### PR DESCRIPTION
**Pull request by Dawars**
This commit (https://github.com/tensorflow/tensorboard/pull/2373) broke compatibility with the mesh plugin.

This PR contains the obvious modifications for now

### CODE REVIEW

File: torch/utils/tensorboard/summary.py [hunk 1]
```diff
@@ -460,7 +460,7 @@ def compute_curve(labels, predictions, num_thresholds=None, weights=None):
     return np.stack((tp, fp, tn, fn, precision, recall))
 
 
-def _get_tensor_summary(name, display_name, description, tensor, content_type, json_config):
+def _get_tensor_summary(name, display_name, description, tensor, content_type, components, json_config, collections):
```
Comment (VitalyFedyunin) [inline torch/utils/tensorboard/summary.py]: Unused collections?

**Comment by ezyang** [comment]
Is this PR still intended to be draft?

**Comment by Dawars** [comment]
Yes, I'm still working on it. This didn't solve the issue just yet

**Comment by orionr** [comment]
cc @sanekmelnikov

**Comment by orionr** [comment]
@Dawars we might have a fix for this at #24039 - looks like a very similar fix.

**Comment by Dawars** [comment]
Great, with tests too, I'm closing this.

**Pull request closed without merging**